README.md

```diff
@@ -203,34 +203,119 @@ All official releases shall be in <code>master</code>. Any updates in between (i
 After a pull request has been open for over 30 days with no activity or response from the author, it'll be automatically marked as stale. We might fork your changes and merge the changes ourselves. Since GitHub tracks contributions by commits, you will be credited.
 </p>
 
-<h2 id="build-yourself">Go Build Yourself</h2>
+<h2 id="building-devicon">Building Devicon</h2>
 <p>
-    Feel free to follow these steps when you want to build the font
-    by yourself.
+Follow these steps to build the website and icons either locally or using <a href=https://www.gitpod.io>Gitpod.io</a>.
 </p>
-<h5>Prerequisites</h5>
-<p>Install <code>gulp</code> (and <code>gulp</code> plugins)</p>
+
+<h3>Table of contents</h3>
+<ol>
+    <li><a href="#using-gitpod">Using Gitpod to build</a></li>
+    <li><a href="#installation">Installation (Linux/Windows)</a></li>
+    <li><a href="#building-icons">Build the icons</a></li>
+    <li><a href="#build-css">Build the CSS stylesheet</a></li>
+    <li><a href="#web-server">Setting up the web server</a></li>
+</ol>
+
+<h3 id="using-gitpod">Using Gitpod.io</h3>
+<p>By using <a href=https://www.gitpod.io)>Gitpod.io</a>, you can easily build the icons and install the<br>required dependencies in one single click. No extra setup is required.</p>
+
+<a href=https://gitpod.io/#https://github.com/devicons/devicon/tree/develop><img src=https://gitpod.io/button/open-in-gitpod.svg alt="Open in Gitpod"></img></a>
+
+<h3 id="installation">Installation</h3>
+
+<p><a href=https://github.com/devicons/devicon/fork>Fork</a> the repository and clone the forked repository.</p>
+
+```bash
+git clone https://github.com/<your-github-username>/devicon.git
+```
+
+<p>In case you don't have Git installed, check the <a href="https://git-scm.com/book/en/v2/Getting-Started-Installing-Git">official guide</a> to install Git on your operating system.</p>
+
+<h4>Install all the necessary NPM dependencies</h4>
+
+<p>In case you don't have NPM installed, check this <a href=https://kinsta.com/blog/how-to-install-node-js/><b>ultimate guide</b></a> on installing Node.js and NPM. These tools are required in order to build Devicon properly.<p>
 
 ```bash
 npm install
 ```
 
-<h5>Build the font and export stylesheet</h5>
-Open <a href="https://icomoon.io/app/#/select">icomoon.io</a> and import <a href="/icomoon.json">icomoon.json</a>. Choose <i>yes</i> when being asked
-if you would like to restore the settings stored in the configuration file.
+<h4>Installing Mozilla Firefox</h4>
```

Comment:
Is mozilla firefox needed? Won't it work with Chrome for example?

Reply:
bump

Reply:
> Is mozilla firefox needed? Won't it work with Chrome for example?

Nope, Mozilla Firefox is needed for this.
I tried it without Firefox, but it didn't work.